### PR TITLE
layers: Fix Clang warning.

### PR DESCRIPTION
**Pull request by null77**
error: using namespace directive in global context in header
error: comparison of integers of different signs: 'uint32_t' (aka 'unsigned int') and 'int'

PTAL.. some warnings I fixed as I was integrating the Clang build with ANGLE/Chromium.

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -11778,10 +11778,11 @@ static bool PreCallValidateCreateSwapchainKHR(layer_data *dev_data, VkSwapchainC
 
         // Validate pCreateInfo->imageExtent against
         // VkSurfaceCapabilitiesKHR::{current|min|max}ImageExtent:
-        if ((capabilities.currentExtent.width == -1) && ((pCreateInfo->imageExtent.width < capabilities.minImageExtent.width) ||
-                                                         (pCreateInfo->imageExtent.width > capabilities.maxImageExtent.width) ||
-                                                         (pCreateInfo->imageExtent.height < capabilities.minImageExtent.height) ||
-                                                         (pCreateInfo->imageExtent.height > capabilities.maxImageExtent.height))) {
+        if ((capabilities.currentExtent.width == std::numeric_limits<uint32_t>::max()) &&
```
Comment (mark-lunarg) [inline layers/core_validation.cpp]: This gives me the shivers.  Using -1 is just as bad, as the spec defines the value specifically as '0xFFFFFFFF'.  Can we use VK_WHOLE_SIZE or one of the other header defines instead?  If not, maybe make this a descriptive static const someplace and use it here and below?

File: loader/vk_loader_platform.h [hunk 2]
```diff
@@ -234,7 +234,6 @@ loader_platform_thread_cond_broadcast(loader_platform_thread_cond *pCond) {
 #ifdef __cplusplus
 #include <iostream>
 #include <string>
-using namespace std;
 #endif // __cplusplus
```
Comment (mark-lunarg) [inline loader/vk_loader_platform.h]: Break this out into a separate 'loader' commit and change the other one to be just 'layers'.

Comment (null77) [inline loader/vk_loader_platform.h]: Done.. I put this in #1274.

**Comment by null77** [comment]
Please take another look.

**Comment by mark-lunarg** [comment]
Pushed to Master.

**Comment by null77** [comment]
Thanks!

**Pull request closed without merging**